### PR TITLE
Flutter Forward 2023 Update

### PR DESCRIPTION
**Pull request by nohe427**
This adds in the code that we would normally live code so it can quickly be uncommented by users during the workshop on Wednesday

*Replace this paragraph with a description of what this PR is changing or adding, and why. Consider including before/after screenshots.*

*List which issues are fixed by this PR. For larger changes, raising an issue first helps
reduce redundant work.*

## Pre-launch Checklist

- [ ] I read the [Effective Dart: Style] _recently_, and have followed its advice.
- [ ] I signed the [CLA].
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-devrel channel on [Discord].

This adds in some code that will make the live coding part easier for flutter forward attendees on Wednesday.

[Effective Dart: Style]: https://dart.dev/guides/language/effective-dart/style
[CLA]: https://cla.developers.google.com/
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Comment by nohe427** [comment]
@domesticmouse - Hey Brett, @puf and I were discussing and we think that having commented out code in the codelab may help the workshop participants complete the codelab as we don't have a large amount of time. Could we add this in temporarily for the workshop and remove it afterwards?

**Comment by domesticmouse** [comment]
Hey @nohe427 there is a conflict in this PR that needs to be rectified before it can be landed.

@miquelbeltran can we land something like this change just for Flutter Forward and not totally trash your work? =)

File: firebase-get-to-know-flutter/step_09/pubspec.yaml [hunk 1]
```diff
@@ -42,6 +42,7 @@ dependencies:
   firebase_core: ^2.3.0
   provider: ^6.0.4
   firebase_ui_auth: ^1.1.0
+  go_router: ^6.0.1
```
Comment (miquelbeltran) [inline firebase-get-to-know-flutter/step_09/pubspec.yaml:45]: I think this is not needed at the moment, but doesn't hurt either ;)

**Comment by miquelbeltran** [comment]
> miquelbeltran can we land something like this change just for Flutter Forward and not totally trash your work? =)

Absolutely no problem, go wild, I will fix afterwards whatever is needed

File: firebase-get-to-know-flutter/step_09/lib/main.dart [hunk 2]
```diff
@@ -6,6 +6,7 @@ import 'package:firebase_auth/firebase_auth.dart'
 import 'package:firebase_core/firebase_core.dart';
 import 'package:firebase_ui_auth/firebase_ui_auth.dart';
 import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
```
Comment (miquelbeltran) [inline firebase-get-to-know-flutter/step_09/lib/main.dart:9]: This is causing the analyzer to fail. The go_router dependency is currently not needed on `main`, go_router stuff will be re-added here: https://github.com/flutter/codelabs/pull/1366

**Review by miquelbeltran — APPROVED** [review]

File: firebase-get-to-know-flutter/step_09/lib/main.dart (hunk 2 above)
Comment (nohe427) [reply firebase-get-to-know-flutter/step_09/lib/main.dart]: Yes, you are correct. Its one of the things that slipped by me after reading it a bit more. Thanks for the quick review

**Review by domesticmouse — APPROVED** [review]

**Comment by nohe427** [comment]
Thanks!

**Pull request merged**